kitops/modelkit/kitfile.py

```diff
@@ -359,14 +188,14 @@ def print(self) -> None:
             None
         """
         warn(
-            "Kitfile.print() is going to be deprecated. " \
+            "Kitfile.print() is going to be deprecated. "
             "To print Kitfile use to_yaml() and your favorite way to log/print; date=2025-02-21",
             DeprecationWarning,
             stacklevel=2,
         )
         print("\n\nKitfile Contents...")
         print("===================\n")
-        output = self.to_yaml()
+        output: str = self.to_yaml()
```

Comment:
If mypy has a hard time with it, let's update it. Otherwise, let's let mypy interpret the type.

Reply:
Gone!